apis/v1alpha1/dcgmexpoter_types.go

```diff
@@ -43,6 +43,10 @@ type DcgmExporterSpec struct {
 	// consumed in the config file for the Collector.
 	// +optional
 	Env []v1.EnvVar `json:"env,omitempty"`
+	// Toleration to schedule OpenTelemetry Collector pods.
```

Comment:
This should say "DCGM Exporter pods" instead of "OpenTelemetry Collector pods".

Reply:
Will fix.